packages/api-plugin-accounts/src/resolvers/Query/filterSearchAccounts.js

```diff
@@ -0,0 +1,31 @@
+import getPaginatedResponse from "@reactioncommerce/api-utils/graphql/getPaginatedResponse.js";
+import wasFieldRequested from "@reactioncommerce/api-utils/graphql/wasFieldRequested.js";
+
+/**
+ * @name Query/accounts
+ * @method
+ * @memberof Accounts/Query
+ * @summary Query for a list of accounts
+ * @param {Object} _ - unused
+ * @param {Object} args - an object of all arguments that were sent by the client
+ * @param {String} args.shopId - id of shop to query
+ * @param {Object} args.conditions - object containing the filter conditions
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} info Info about the GraphQL request
+ * @returns {Promise<Object>} Accounts
+ */
+export default async function filterSearchAccounts(_, args, context, info) {
```

Comment:
As I have mentioned a couple of times, not sure why we keep on insisting to calling this `filterSearch` when filter would suffice. If we add search functionality this seems like it would be confusing. Why not just call it filter?

Reply:
Renamed all the instances
filterSearchAccounts -> filterAccounts
filterSearchCustomers -> filterCustomers
filterSearchOrders -> filterOrders
filterSearchProducts -> filterProducts